src/Telegram.php

```diff
@@ -830,4 +830,68 @@ public function enableBotan($token, array $options = [])
 
         return $this;
     }
+
+    /**
+     * Run provided commands
+     *
+     * @param array $commands
+     * @param array $update
+     *
+     * @throws TelegramException
+     */
+    public function runCommands($commands, array $update = [])
+    {
+        if (!is_array($commands)) {
+            throw new TelegramException('No command(s) provided!');
+        }
+
+        $this->botan_enabled = false;   // Force disable Botan.io integration, we don't want to track self-executed commands!
+
+        $result = Request::getMe()->getResult();
+
+        if (!$result) {
+            throw new TelegramException('Received invalid getMe result!');
+        }
+
+        $bot_id   = $result->getId();
+        $bot_name = $result->getFirstName();
+
+        $update_template = [
+            'update_id' => 0,
+            'message' => [
+                'message_id' => 0,
+                'from' => [
+                    'id' => $bot_id,
+                    'first_name' => $bot_name
+                ],
+                'date' => time(),
+                'chat' => [
+                    'id' => $bot_id,
+                    'type' => 'private',
+                ],
+                'text' => ''
+            ]
+        ];
+
+        $update = array_merge($update_template, $update);
+
+        $this->enableAdmin($bot_id);    // Give bot access to admin commands
+        $this->getCommandsList();       // Load full commands list
+
+        foreach ($commands as $command => $parameter) {
+            if (is_numeric($command)) {     // if array/key is not associative $command will be integer and $parameter will be the actual command!
```

Comment:
Just so I understand this correctly:
```php
$commands = [
    'whoami',
    'echo' => 'hello world!',
];
```
Would call `/whoami` and `/echo hello world!`, right?

If yes, I really like your thinking behind this, although this does limit to 1 single call of any single command with params, as the key is unique.
```php
$commands = [
    'echo' => 'will I be echoed?',
    'echo' => 'nope, just me!',
];
```

Might be an idea to just add the whole command as array elements, like so:
```php
$commands = [
    '/echo will I be echoed?',
    '/echo yes! and so will I :-)',    
];
```
This way, also "normal" text commands could be sent theoretically, although the way it's coded now this won't work.